addons/im_livechat/static/src/new/core/thread_model.js

```diff
@@ -15,4 +15,25 @@ patch(Thread.prototype, "im_livechat", {
     get isChatChannel() {
         return this.type === "livechat" || this._super();
     },
+
+    get allowMarkAsRead() {
```

Comment:
I would change this name, it is used in the context of last seen message, current name is too close to inbox mark as read.